Pipfile

```diff
@@ -20,7 +20,7 @@ graphviz = "<=0.18.2"
 requests = "<=2.26.0"
 pandas = "<=1.3.4"
 plotly = "<=5.4.0"
-prompt-toolkit = "==3.0.19"
+prompt-toolkit = "<=3.0.36"
```

Comment:
Because ipython is updated to 8.9.0 and it requires prompt-toolkit>=3.0.30, I changed the max version of prompt-toolkit to the latest 3.0.36. Also, the problem of #186 has been fixed.